src/main/scala/org/virtuslab/internal/load/reader/YamlReader.scala

```diff
@@ -94,34 +132,49 @@ class YamlReader(in: CharSequence) extends Reader {
         skipCharacter()
       case _ => ()
 
-    skipUntilNextToken()
     val foldedIndent = indent
+    parseBlockHeader()
+    skipUntilNextIndent(foldedIndent)
+
+    def chompedEmptyLines() =
+      while (peekNext() == Some('\n')) {
+        skipCharacter()
+        sb.append("\\n")
+      }
+
+      skipCharacter()
```

Comment:
Why additional `skipCharacter`? `skipUntilNextIndent` isn't enough?

Reply:
No, we have to also skip new line.